loader.tf

```diff
@@ -36,3 +36,59 @@ resource "aws_s3_bucket_notification" "notifications" {
     filter_prefix       = "full"
   }
 }
+
+resource "aws_sns_topic" "success_sns_topic" {
+  name = var.success_topic_name
+  policy = data.aws_iam_policy_document.success_sns_notification_policy.json
+}
+
+resource "aws_sns_topic" "failure_sns_topic" {
+  name = var.failure_topic_name
+  policy = data.aws_iam_policy_document.failure_sns_notification_policy.json
+}
+
```

Comment:
Is there a reason not to have a single policy with two resources in it? Othewise these two polices are identical.

Reply:
Good point. I've changed it to use just one policy with two resources.